package/lib/index.js

```diff
@@ -29,6 +29,15 @@
 //                                                                                  //
 // //////////////////////////////////////////////////////////////////////////////// //
 
+/**
+ * @private
+ */
+function suggest (incorrectName, correctName) {
+  return function() {
```

Comment:
The CI build failed because there was a lint error:

`/home/travis/build/jankapunkt/js-set-extension/package/lib/index.js:36:18: Missing space before function parentheses.`

was the output. Should be easy to fix using `npm run lint-fix`